express/parsers/__init__.py

```diff
@@ -10,6 +12,11 @@ def __init__(self, *args, **kwargs):
         self.args = args
         self.kwargs = kwargs
 
+    def unix_path_names(self, full_path: str) -> Tuple[str, str]:
```

Comment:
Nit: our convention would be to either use `get_` in the name or define a function as @property, so maybe `get_unix_path_names`?

Reply:
Updated for consistency